adapter/src/hermes/adapter/constants.h

```diff
@@ -28,4 +28,29 @@ const char* kHermesConf = "HERMES_CONF";
  * from I/O. This is needed as we should not intercept these files.
  */
 const char* kHermesExtension = ".hermes";
+/**
+ * kHermesBufferMode env variable is used to define the mode in which Hermes'
+ * STDIO adapter operates.
+ *
+ * It supports 3 Values: PERSISTENT/BYPASS/SCRATCH
+ */
+const char* kAdapterMode = "ADAPTER_MODE";
+/**
+ * kHermesBufferModeInfo env variable is used to define the additional
+ * information for the selected mode @see kHermesBufferMode.
```

Comment:
This documentation appears to reference old names that were changed.

Reply:
FIXED.